Wizardry/Core/UnitStatusGetter/source/MovGetter.c

```diff
@@ -68,5 +68,10 @@ int MovGetterSkills(int status, struct Unit * unit)
     }
 #endif
 
+#if defined(SID_Poise) && (COMMON_SKILL_VALID(SID_Poise))
+    if (SkillTester(unit, SID_Poise))
+            status += 1;
```

Comment:
lol

Reply:
Wasn't my idea to be fair 😛

Some skills are just odd like that